R/uk.R

```diff
@@ -131,22 +131,26 @@ To get hospital admissions data, include argument 'nhsregions = TRUE'. This retu
 }
 
 
-#' UK Regional Daily COVID-19 Count Data - LTLA
+#' UK Regional Daily COVID-19 Count Data - Local Authority level
 #'
-#' @description Extracts daily COVID-19 data for the UK, stratified by Upper Tier Local Authority 
+#' @description Extracts daily COVID-19 data for the UK, stratified by Local Authority
 #' Data source:
+#' @param resolution "utla" (default) or "ltla", depending on which geographical resolution is preferred
 #' @inheritParams get_uk_data
 #' @return A data frame of daily COVID cases for the UK by region, to be further processed by \code{get_regional_data()}.
 #' @importFrom dplyr bind_rows mutate rename select left_join %>%
 #' @importFrom stringr str_detect
 #' @importFrom lubridate ymd
 #' 
-get_uk_regional_cases_with_level_2 <- function(release_date = NULL) {
+get_uk_regional_cases_with_level_2 <- function(resolution = c("utla", "ltla"), release_date = NULL) {
```

Comment:
This needs to default to only one input and have the choices checked in `match.arg`?

Reply:
7d2fb1e should fix it I think